dashboard/app/models/peer_review.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: peer_reviews
+#
+#  id              :integer          not null, primary key
+#  user_id         :integer
+#  from_instructor :boolean          default(FALSE), not null
+#  script_id       :integer          not null
+#  level_id        :integer          not null
+#  level_source_id :integer          not null
+#  data            :text(65535)
+#  status          :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+# Indexes
+#
+#  index_peer_reviews_on_level_id         (level_id)
+#  index_peer_reviews_on_level_source_id  (level_source_id)
```

Comment:
This ends up creating an ugly `fk_rails_afcb8b7aac` foreign key when not indexed.  I'm going to update the indexes (and combine indexes where needed) after I see how we'll be typically querying the table.

One place we may want the `level_source_id` index would be finding the list of reviews for the most recent version of a submission.